enterprise/ha/src/test/java/org/neo4j/ha/ClusterTransactionIT.java

```diff
@@ -55,8 +55,8 @@
     public void setUp() throws Exception
     {
         cluster = clusterRule.withProvider( clusterOfSize( 3 ) )
-                             .withSharedSetting( HaSettings.ha_server, ":6001-6005" )
-                             .withSharedSetting( HaSettings.tx_push_factor, "2" ).startCluster();
+                .withSharedSetting( HaSettings.ha_server, ":6001-6005" )
```

Comment:
If we can avoid checkstyling in maint branches that'd be grand... never mind this change though.